src/towncrier/newsfragments/391.bugfix

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I think that we have 2 separate issues here.

* The detection of duplicate release notes was fixed and SOME PREVIOUS BEHAVIOUR is no longer triggered.
* Support for having the release notes for each version in a separate file is working again. This is regression introduced in VERSION X.Y.Z.